2023/gimmecloudtalks/data.json

```diff
@@ -0,0 +1,30 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "",
```

Comment:
We need a GitHub handle in here - for you that would be dwnatwick
e.g. 
"GitHubHandles": [ "dwnatwick" ]